test/e2e_node/remote/utils.go

```diff
@@ -43,6 +60,19 @@ func installCNI(host, workspace string) error {
 	if output, err := SSH(host, "sh", "-c", cmd); err != nil {
 		return fmt.Errorf("failed to install cni plugin on %q: %v output: %q", host, err, output)
 	}
+
+	// The added CNI network config is not needed for kubenet. It is only
+	// used when testing the CNI network plugin, but is added in both cases
+	// for consistency and simplicity.
+	glog.V(2).Infof("Adding CNI configuration on %q", host)
+	cniConfigPath := filepath.Join(workspace, cniConfDirectory)
+	cmd = getSSHCommand(" ; ",
+		fmt.Sprintf("mkdir -p %s", cniConfigPath),
+		fmt.Sprintf("echo %s > %s", quote(cniConfig), filepath.Join(cniConfigPath, "mynet.conf")),
```

Comment:
Also, today the cni binaries are pulled down from  https://storage.googleapis.com/kubernetes-release/network-plugins/cni-" + cniRelease + ".tar.gz.  I figured there was a good reason for this, maybe @freehan knows better than I.